watchman.c

```diff
@@ -179,6 +179,32 @@ int watchman_pclose(struct watchman_error *error, struct watchman_popen *popen)
     }
 }
 
+/**
+ * Call select() on the given fd until it's ready to read from or
+ * until the timeout expires. Updates timeout to indicate the
```

Comment:
nit: the first sentence is in imperative mood; the second is in indicative.  Should be "Update"